pkg/token/execCredentialPlugin.go

```diff
@@ -29,6 +31,14 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	var execCredential clientauthentication.ExecCredential
+	if err := json.Unmarshal([]byte(env), &execCredential); err != nil {
```

Comment:
Is it now required to specify `KUBERNETES_EXEC_INFO`? My workflow for v0.0.20 now throws `Error: cannot convert to ExecCredential: unexpected end of JSON input`. It seems perhaps because `env` here is `""` which then can't be unmarshalled into a `clientauthentication.ExecCredential`.